.travis.yml

```diff
@@ -1,60 +1,74 @@
 language: go
-sudo: required
 dist: bionic
 
 go:
   - 1.13.x
   - 1.14.x
 
+os: linux
+
+arch:
+  - amd64
+  - arm64
+  - ppc64le
+
 env:
   global:
   - PATH=$GOROOT/bin:$GOPATH/bin:$PATH
   - CGO_ENABLED=0
-  matrix:
-  # note: arm64 is tested below
-  - TARGET=386
-  - TARGET=amd64
-  - TARGET=arm
-  - TARGET=ppc64le
-  - TARGET=s390x
-  - TARGET=mips64le
-
-matrix:
+
+jobs:
   fast_finish: true
   include:
     - os: windows
-      env: TARGET=amd64
+      arch: amd64
       go: 1.13.x
     - os: windows
-      env: TARGET=amd64
+      arch: amd64
       go: 1.14.x
-    - os: linux
-      env: TARGET=arm64
+    - name: "cross-compile other architectures, go 1.13"
+      env: "TARGET='386 arm s390x mips64le ppc64le'"
       go: 1.13.x
-      arch: arm64
-    - os: linux
-      env: TARGET=arm64
+      os: linux
+      arch: amd64
+    - name: "cross-compile other architectures, go 1.14"
+      env: "TARGET='386 arm s390x mips64le ppc64le'"
       go: 1.14.x
-      arch: arm64
+      os: linux
+      arch: amd64
 
 install:
   - go get github.com/onsi/ginkgo/ginkgo
   - go get github.com/containernetworking/cni/cnitool
   - go get golang.org/x/tools/cmd/cover
   - go get github.com/modocache/gover
   - go get github.com/mattn/goveralls
-  - go mod vendor
+  # installing the tools has changed our vendoring; undo that
+  - git checkout -- go.mod go.sum
```

Comment:
Ick.  Can't you put what you need in the `require` section so it will come out the same?

Reply:
maybe, maybe not; there's no guarantee that these `go get` commands won't change vendor in the future.

The problem is we're using `go get` as both a library manager and a packaging system for installing CLI tools. And `go get` makes it hard to do the latter in a sane way.